api/src/services/script-runner.service.ts

```diff
@@ -605,6 +616,152 @@ export class ScriptRunnerService {
     return { success: true };
   }
 
+  /**
+   *
+   * @param req incoming request object
+   * @param dataTransferDTO data transfer endpoint args. Should contain foreign db connection string
+   * @returns successDTO
+   * @description transfers assets for listings in the specified space into the new space
+   */
+  async transferListingAssetData(
+    req: ExpressRequest,
+    dataTransferDTO: AssetTransferDTO,
+    prisma?: PrismaClient,
+  ): Promise<SuccessDTO> {
+    // script runner standard start up
+    const requestingUser = mapTo(User, req['user']);
+    await this.markScriptAsRunStart(
+      `data transfer assets ${dataTransferDTO.jurisdiction}`,
```

Comment:
issue: when I try run the second page it fails because the script has already run. Should the page be appended to the end of the name?